packages/create-mc-app/bin/create-mc-app.js

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-console, prefer-object-spread/prefer-object-spread */
+
+const path = require('path');
+const fs = require('fs');
+const os = require('os');
+const execSync = require('child_process').execSync;
+const mri = require('mri');
+
+const flags = mri(process.argv.slice(2), { alias: { help: ['h'] } });
+const commands = flags._;
+
+const availableTemplates = ['starter'];
```

Comment:
Just curious: what are further templates? Given we have this option we already have some in the back of our heads?

Reply:
Nothing specific at the moment but I can image having different templates focusing on different parts. We can also have templates for e.g. Typescript, Reason, Parcel, etc.

Reply:
Got it.